deploy/charts/llmos-operator/templates/rbac.yaml

```
@@ -12,3 +12,27 @@ subjects:
   - kind: ServiceAccount
     name: {{ include "llmos-operator.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name}}-registry-reader
+rules:
+- apiGroups:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)